ajax/libs/FileSaver.js/package.json

```diff
@@ -11,5 +11,13 @@
     "type": "git",
     "url": "https://github.com/eligrey/FileSaver.js"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "npmName": "file-saver",
+  "npmFileMap": [
+    {
+      "files": [
+        "*.js"
```

Comment:
@bdukes thanks for your PR, I prefer to use `FileSaver*` here to be more complex, would you please update the expression here? Thanks.